SPECS/shadow-utils/login-defs

```diff
@@ -190,7 +190,9 @@ KILLCHAR        025
 # 022 is the default value, but 027, or even 077, could be considered
 # for increased privacy. There is no One True Answer here: each sysadmin
 # must make up their mind.
-UMASK           022
+# Security patch for CCE-14847-8, msid: 29
+# The default umask for all users should be set to 077 in login.defs
+UMASK           077
```

Comment:
I'm worried this may be disruptive to existing  users, we might want to  make t his part of the post processing steps.